<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtendedSearhForm</class>
 <widget class="QWidget" name="ExtendedSearhForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Область:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditArea">
           <property name="text">
            <string>Везде</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Искать:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditFind"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonFind">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSelectArea">
         <property name="text">
          <string>Выбрать область...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxReg">
        <property name="text">
         <string>Учитывать регистр</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxOnlyWord">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Только слова целиком</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxComment">
        <property name="text">
         <string>Поиск в комментариях</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBoxNamePIC">
        <property name="text">
         <string>Поиск по имени ПИВ(если имя было указано в описании данных) </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxName">
        <property name="text">
         <string>Поиск по имени</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTreeWidget" name="treeWidgetResult">
     <column>
      <property name="text">
       <string>Слово</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Путь</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ExtendedSearhForm</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditFind</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonFind</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
